### ISSUE TITLE
v1.8.2: -workdir argument ignored

### ISSUE THREAD

**Issue by ivdok** [telegramdesktop/tdesktop#6449]
### Steps to reproduce
1. Install telegram-desktop-bin from AUR
2. Start first instance with default desktop launcher.
3. Start another instance via 'telegram-desktop -startintray -many -workdir /home/ivdok/.local/share/Telegram2'.
4. Terminate all instances, start only another profile with command provided above. Same outcome.

### Expected behaviour
Second instance starts in tray, and in parallel to original instance

### Actual behaviour
Only original profile, made via desktop launcher, works.

### Configuration
**Operating system:** Arch Linux current

**Version of Telegram Desktop:** 1008002

**Used theme**: Matrix

<details><summary><b>Logs</b>:</summary>

```
[2019.08.28 08:59:41] Launched version: 1008002, install beta: [FALSE], alpha: 0, debug mode: [FALSE], test dc: [FALSE]
[2019.08.28 08:59:41] Executable dir: /usr/bin/, name: telegram-desktop
[2019.08.28 08:59:41] Initial working dir: /home/ivdok/
[2019.08.28 08:59:41] Working dir: /home/ivdok/.local/share/TelegramDesktop/
[2019.08.28 08:59:41] Command line: telegram-desktop -startintray -many -workdir /home/ivdok/.local/share/Telegram2
[2019.08.28 08:59:41] Executable path before check: /usr/bin/telegram-desktop
[2019.08.28 08:59:41] Logs started
[2019.08.28 08:59:41] Fontconfig version: 2.13.1.
[2019.08.28 08:59:41] Custom FONTCONFIG_FILE: /home/ivdok/.local/share/TelegramDesktop/tdata/fc-custom-1.conf
[2019.08.28 08:59:42] Many instance allowed, starting...
[2019.08.28 08:59:42] Primary screen DPI: 96.0046
[2019.08.28 08:59:42] Font Error: could not resolve 'Open Sans Semibold' font, got 'Open Sans'.
[2019.08.28 08:59:42] Could not load 'ayatana-appindicator3' with version 1 :(
[2019.08.28 08:59:42] Could not load 'appindicator3' with version 1 :(
[2019.08.28 08:59:42] Could not load 'ayatana-appindicator' with version 1 :(
[2019.08.28 08:59:42] Could not load 'appindicator' with version 1 :(
[2019.08.28 08:59:42] Error: failed to load 'gdk_x11_drawable_get_xdisplay' function!
[2019.08.28 08:59:42] Getting DE, XDG_CURRENT_DESKTOP: 'XFCE'
[2019.08.28 08:59:42] Getting DE, DESKTOP_SESSION: 'xfce'
[2019.08.28 08:59:42] Getting DE, KDE_SESSION_VERSION: ''
[2019.08.28 08:59:42] DE: XFCE
[2019.08.28 08:59:42] Tray Icon: Try Qt = [TRUE], Prefer appindicator = [TRUE]
[2019.08.28 08:59:42] App Info: reading settings...
[2019.08.28 08:59:42] App Info: reading encrypted settings...
[2019.08.28 08:59:42] Lang Info: Loaded cached, keys: 2398
[2019.08.28 08:59:42] Lang Error: Unexpected tag (key 'lng_call_box_status_group:count')
[2019.08.28 08:59:42] Lang Error: Unknown tag (key 'lng_manage_discussion_group_warning:visible')
[2019.08.28 08:59:42] OpenAL Logging Level: (not set)
[2019.08.28 08:59:42] Audio Playback Devices: OpenAL Soft
[2019.08.28 08:59:42] Audio Playback Default Device: OpenAL Soft
[2019.08.28 08:59:42] Audio Capture Devices: Built-in Audio Analog Stereo;Monitor of Built-in Audio Analog Stereo
[2019.08.28 08:59:42] Audio Capture Default Device: Built-in Audio Analog Stereo
[2019.08.28 08:59:42] App Info: reading map...
[2019.08.28 08:59:42] App Info: reading encrypted map...
[2019.08.28 08:59:42] App Info: reading encrypted user settings...
[2019.08.28 08:59:42] App Info: encrypted user settings read.
[2019.08.28 08:59:42] App Info: reading encrypted mtp data...
[2019.08.28 08:59:42] MTP Info: read keys, current: 3, to destroy: 0
[2019.08.28 08:59:42] Map read time: 10
[2019.08.28 08:59:42] TCP Error: network - Network unreachable
[2019.08.28 08:59:42] HTTP Error: network error 99 - Network unreachable
[2019.08.28 08:59:42] Export Info: Destroy top bar by controller removal.
[2019.08.28 08:59:42] Using Qt tray icon.
[2019.08.28 08:59:42] Tray Icon: Using Qt tray icon, available: [TRUE]
[2019.08.28 08:59:42] Not using Unity Launcher count.
[2019.08.28 08:59:43] TCP Error: network - Network unreachable
[2019.08.28 08:59:43] HTTP Error: network error 99 - Network unreachable
```
</details>

**Additional comment**: This is probably a regression of #4123.

**Comment by ivdok** [comment]
[ivdok@ThinkPad-t440s ~]$ telegram-desktop -startintray  -many -workdir "/home/ivdok/.local/share/TelegramHome"
Qt: Session management error: Authentication Rejected, reason : None of the authentication protocols specified are supported and host-based authentication failed

**Comment by ivdok** [comment]
Using `telegram-desktop` from Arch Linux instead of upstream also doesn't net any gains - while at least -many executes, it still uses my original workdir.

**Comment by Aokromes** [comment]
this bug is still valid?

**Comment by Aokromes** [comment]
no answer after near 1 month, if this bug is still valid, open a new ticket with one updated telegram version downloaded from desktop.telegram.org or github.com
